crates/ruff_benchmark/benches/ty.rs

```diff
@@ -466,6 +463,56 @@ fn benchmark_complex_constrained_attributes_2(criterion: &mut Criterion) {
                                 return
                             if isinstance(self.b, str):
                                 return
+                            if isinstance(self.b, str):
+                                return
+                            if isinstance(self.b, str):
+                                return
```

Comment:
We reduced this so much that it wasn't actually showing a problematic runtime anymore. So I added two more clauses.